### ISSUE TITLE
[PC desktop][Word] When "Shared runtime" requirement set the "Custom tab" is not displayed in the Word desktop, though in Word online it shows ups fine

### ISSUE THREAD

**Issue by svehera-nd** [OfficeDev/office-js#5228]
# Provide required information needed to triage your issue
1. Use the manifest from the file below  (wordIssue.txt)
2. Install it for word desktop
3. Open Word desktop 

[wordIssue.txt](https://github.com/user-attachments/files/18253610/wordIssue.txt)

You can also try to install a sample from documentation, it works fine and custom tab is displayed after the addin is installed
[ms-sample.txt](https://github.com/user-attachments/files/18253635/ms-sample.txt)

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: **PC desktop**
* Host [Excel, Word, PowerPoint, etc.]: **Word**
* Office version number:  Word for Microsoft 365 MSO (Version 2411 Build 16.0.18227.20082) 64-bit 
* Operating System: Windows 10 Pro 22H2 19045.5247
 
## Expected behavior
Custom tab shows up in Word desktop

## Current behavior
Custom tab does NOT show up in Word desktop but shows up in Word online for the same manifest

## Steps to reproduce
1. Use the manifest from the file below
2. Install it for word desktop
3. Open Word desktop 

## Link to live example(s)
- NA

# Provide additional details
- NA

## Context
- NA

## Useful logs
- NA

**Comment by RuizhiSunMS** [comment]
Hi @svehera-nd, thx for reaching out here. I can repro this. Put it as #9650724 into our backlog. I will move it to owning team for further investigation. Will reply here if any update.

**Comment by svehera-nd** [comment]
@RuizhiSunMS  Thanks

**Comment by xiruatms** [comment]
@svehera-nd , thanks for reporting this. 

I noticed that: in CustomTab, there's in spaces in id field.

![Image](https://github.com/user-attachments/assets/ce95d648-ee82-4bc2-a7f1-5ac3eda1586d)

After removing those spaces, "My custom tab" can be shown properly.

![Image](https://github.com/user-attachments/assets/a7581f55-085d-4e7d-a1ef-5629029b2dde)

**Comment by svehera-nd** [comment]
Yeah probably it was that. I didn't notice but we somehow got it fixed
